src/tools/adig.c

```diff
@@ -251,6 +262,80 @@ static ares_bool_t read_cmdline(int argc, const char * const *argv,
   return ARES_TRUE;
 }
 
+static ares_bool_t read_rcfile(adig_config_t *config)
```

Comment:
in read_cmdline() should probably skip config->args_processed assignment and sanity check if is_rcfile

Reply:
Added.